storage/postgresql/pgbench_test_README.md

```diff
@@ -0,0 +1,98 @@
+### pgbench load tool for OCP pods 
+
+
+[pgbench_test.sh](https://github.com/ekuric/openshift/blob/master/postgresql/pgbench_test.sh) script can be used for executing pgbench benchmark tool
+inside postgresql pod running on top of Kubernetes / OpenShift Container Platform (OCP) when storage can be allocated 
+dynamically using storage classes
+
+### Supported options 
+
+pgbench_test.sh supports below options 
+
+``` 
+The following options are available:
+
+		-n --namespace - name for new namespace to create pod inside
+		-t --transactions - the number pgbench transactions
+		   --scaling - pgbench scaling factor
+		-e --template -  what template to use
+		-v --volsize - size of volume for database
+		-m --memsize - size of memory to assign to postgresql pod
+		-i --iterations - how many iterations of test to execute
+		 --mode - what mode to run: cnsfile or cnsblock, or otherstorage
+		-r --resultdir - name of directory where to place pgbench results
+		 --clients - number of pgbench clients
+		 --threads - number of pgbench threads
+		 --storageclass - name of storageclass to use to allocate storage
+
+``` 
+
+### Setup
+
+pgbench_test.sh exepects below to be in place and functioning before executing it
+
+- when executed as standalone script pgbench_test.sh does not require any prerequest 
+For this test case, check below hot to run it
+
+- when used as input script for [pbench](https://github.com/distributed-system-analysis/pbench) pbench-user-benchmark script
+
+For pbench test case, it is necessary to have below in order to make it work 
+
+- installed and properly setup pbench from [pbench](https://github.com/distributed-system-analysis/pbench)
+- installed pgbench ( on centos/rhel it is part of `postgresql-contrib package`)
+- template which supports dynamic storage provision using storage classes
+
+### Usage:  
+
+- standalone case 
+
+```
+./pgbench_test.sh -n <namespace> -t <transactions> -e <template> -v <vgsize> -m <memsize> -i <iterations> --mode <mode> -r resultdir --clients <number of clients> -- threads <number of threads> --storageclass <name of storageclass>
```

Comment:
--scaling is missing?

Reply:
seems I sent wrong version... will fix rest